packer/grapl-service/setup_amazon-linux-2.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
We should be consistent with `_` and `-`, and just pick one for the file name.

Reply:
i just deleted it